packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -549,18 +539,27 @@ export class SwapQuoter {
         // get batches of orders from different sources, awaiting sources in parallel
         const orderBatchPromises: Array<Promise<SignedOrder[]>> = [];
         orderBatchPromises.push(this._getSignedOrdersAsync(makerAssetData, takerAssetData)); // order book
+
+        const rfqtOptions = this._rfqtOptions;
+        const quoteRequestor = new QuoteRequestor(
+            rfqtOptions ? rfqtOptions.makerAssetOfferings || {} : {},
+            rfqtOptions ? rfqtOptions.warningLogger : undefined,
+            rfqtOptions ? rfqtOptions.infoLogger : undefined,
+            this.expiryBufferMs,
+        );
+
         if (
             opts.rfqt &&
             opts.rfqt.intentOnFilling &&
             opts.rfqt.apiKey &&
-            this._rfqtTakerApiKeyWhitelist.includes(opts.rfqt.apiKey) &&
-            !(marketOperation === MarketOperation.Buy && this._rfqtSkipBuyRequests)
+            this._rfqtTakerApiKeyWhitelist().includes(opts.rfqt.apiKey) &&
```

Comment:
`this._rfqtTakerApiKeyWhitelist().includes(opts.rfqt.apiKey)` -> `this._isApiKeyWhitelisted(ops.rfqt.apiKey)`

seems like the only usage of it.

Reply:
good call -- 1c84709db